Error pages, titles and remove ordering

nottrobin opened the pull request:
- Fixes up titles
- Added error pages
- Remove any `order_by` logic, as the API should very soon return results in the desired order itself.

QA
--

`./run`, go to http://0.0.0.0:8034/idontexist and see the 404 page.

Seeing the 500 pages is a bit harder. Why not throw a random undefined variable call into a view in `app.py` or something?

nottrobin committed:
Add 404 and 500 pages, and fixup titles

webteam-app commented:
Starting demo at: https://certification-ubuntu-com-canonical-web-and-design-pr-23.run.demo.haus/

nottrobin committed:
Remove stray slashes from URLs

nottrobin committed:
Remove "order_by=make"

Default sorting is coming, and so we should just use the
order in which items are returned from the API.

jpmartinspt reviewed (approved):
Works fine.
